README.md

```diff
@@ -1,2 +1,37 @@
-# intuit-login-page-clone
-This is a clone of intuit website's login page clone
+# intuit-signup-page-clone
+This is a clone of intuit website's signup page clone
+
+## BUILT WITH
+
+- HTML
+
+- CSS
+
+## Live Demo
+- [Live Demo Link](https://adejam.github.io/intuit-signup-page-clone/index.html)
+
+## Images
+![Youtube video page clone](/images/signup-page.png)
+## Screenshot: 
+![Youtube video page clone](/images/signup-page.png)
```

Comment:
[Optional]

Is this not the same, is there a reason why you are repeating the same picture?

I would recommend using only one. Also, the alt should be about the current project not **Youtube video page clone**.

Reply:
Thanks for the review. I'll work on it.